app/src/androidTest/java/ch/epfl/skysync/screens/userManagement/UserManagementTest.kt

```diff
@@ -0,0 +1,87 @@
+package ch.epfl.skysync.screens.userManagement
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTextInput
+import androidx.navigation.compose.rememberNavController
+import ch.epfl.skysync.models.calendar.AvailabilityCalendar
+import ch.epfl.skysync.models.calendar.FlightGroupCalendar
+import ch.epfl.skysync.models.flight.RoleType
+import ch.epfl.skysync.models.user.User
+import ch.epfl.skysync.screens.RoleFilter
+import ch.epfl.skysync.screens.SearchBar
+import ch.epfl.skysync.screens.UserCard
+import ch.epfl.skysync.screens.UserManagementScreen
+import junit.framework.TestCase.assertEquals
+import org.junit.Rule
+import org.junit.Test
+
+class UserManagementTest {
+  @get:Rule val composeTestRule = createComposeRule()
+
+  private val mockUsers =

```

Comment:
You should use DatabaseSetup. There are already users defined

Reply:
Wasn't aware, is fixed now